rsts/concepts/nodes.rst

```diff
@@ -23,7 +23,7 @@ For example, an array task will be enclosed by a single node.
 
 Workflow Nodes
 --------------
-A node can contain an entire sub-workflow. Because workflow executions always require a launch plan, workflow nodes have a reference to a launch plan used to trigger their enclosed workflows.
+A node can contain an entire sub-workflow. Since workflow executions always require a launch plan, workflow nodes can reference a launch plan that would trigger their enclosed workflows.
```

Comment:
```suggestion
A node can contain an entire sub-workflow. Since workflow executions always require a launch plan, workflow nodes have a reference to a launch plan to trigger their enclosed workflows.
```